.changeset/stupid-toes-own.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-css": minor
+---
+
+Moved `iui-fade-in` animation from just being scoped in Dialog-related classes to now be global/unscoped.
```

Comment:
This changeset could be refocused to highlight what matters to the user. (`iui-fade-in` is just an implementation detail)

```suggestion
`iui-backdrop` will now also fade in when `iui-backdrop-visible` is present during mount.
```